old-tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Can we reverse this? Have a `tsconfig.json` and a `new-tsconfig.json`? The new then can be smaller since it only needs to extend the original and add a new option

Reply:
Sure!